bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -85,14 +89,17 @@
     private LedgerMetadataFormat.DigestType digestType;
     private byte[] password;
 
-    private Map<String, byte[]> customMetadata = Maps.newHashMap();
+    private Map<String, byte[]> customMetadata;
 
     public LedgerMetadata(int ensembleSize, int writeQuorumSize, int ackQuorumSize,
```

Comment:
it might be good to consider using builder pattern here. ledger type should have default value.